src/components/IconButton/index.js

```diff
@@ -101,7 +104,7 @@ const styles = {
     }
   }),
   label: css({
-    fontFamily: fontFamilies.sansSerifMedium,
+    fontFamily: fontStyles.sansSerifMedium,
```

Comment:
you need to spread it:
```suggestion
    ...fontStyles.sansSerifMedium,
```
(contains family and weight)